documentation/Functions.md

```diff
@@ -60,3 +68,22 @@ buildJava("cobigenexample", true)
 createFile("cobigenexample/core/src/main/java/com/example/application/cobigenexample/customermanagement/dataaccess/api/CustomerEntity.java", "files/CustomerEntity.java")
 
 ***
+
+### changeFile
+#### parameter 
+1. Path of the file to be changed (relative path to the workspace directory)
+2. Name of the placeholder
+3. Path of the file to get the content from or a string, that should be inserted.
+#### example 
+changeFile("cobigenexample/core/src/main/java/com/example/application/cobigenexample/customermanagement/dataaccess/api/CustomerEntity.java", "//PLACEHOLDER-GETTER-SETTER", {"file": "files/CustomerEntityFunction.java"})
+#### details
+##### Name of the placeholder
+Naming Convention
+[Comment-Syntax]PLACEHOLDER-[ATTRIBUTE]
```

Comment:
Maybe better name the last part 'DESCRIPTION'. The placeholder can stand for anything, not just for attributes